docs/quickstarts/nextjs.mdx

```diff
@@ -214,11 +213,7 @@ export default function Home() {
 ``` 
 </CodeBlockTabs>
 
-Then, visit your app's homepage at [`localhost:3000`](http://localhost:3000) while signed out or signed in to see `<SignInButton />` and `<UserButton />` in action.
-
-### Sign out

```

Comment:
The previous step tells the user to be signed out to see the `<SignInButton />` or sign in to see the `<UserButton />` - so we aren't sure which state they will be in when they move on to the next step. 
We can either
- remove the section
- reword the last "Sign out" step to say "Finally, sign in to your app at localhost:3000 . Use the UserButton by selecting your avatar, then selecting Sign out". but it seems redundant of the last piece of copy in the previous step (the "Then, visit your app's homepage... to see SignInButton and UserButton"

so I'm not sure if we want to keep this last "Sign out" section or remove it altogether ?